Solutions/MimecastSEG/SolutionMetadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "publisherId": "mimecast",
-    "offerId": "azure-sentinel-solution-mimecastseg",
+    "offerId": "mimecastseg",

```

Comment:
Hi @nipun-crestdatasystem, please revert this change in Offer ID, as per documentation the earlier offer ID was correct. 
Refer - https://github.com/Azure/Azure-Sentinel/tree/master/Solutions#author-and-publish-a-solution-offer
Please let us the know the reason behind changing the offer ID if you want the updated offer ID.